.circleci/config.yml

```diff
@@ -57,19 +49,11 @@ jobs:
       - run: |
           apt-get update
           apt-get install -y p7zip-full
-      - restore_cache:
-          keys:
-            - v3-node-{{ checksum "NODE_VERSION.md" }}
       - restore_cache:
           keys:
             - v3-npm-{{checksum "yarn.lock"}}
       - run: yarn install
       - run: yarn pack:tarballs
-      - save_cache:
```

Comment:
@mshanemc why is save-cache being removed? Does this happen elsewhere÷

Reply:
It was cached using a key based on the node_version file (which is now removed).  It should be removed everywhere it was used (they do a restore cache for the yarn lock, which happens in previous jobs and on branches).

Reply:
what's going to happen in real life...node bumps LTS pretty often for patch versions.  So 14.17 today, may very well be 14.18 by next week's release.  We don't want to cache those at all.